setup.py

```diff
@@ -199,6 +200,48 @@ def run(self):
         print('\nAll generate scripts passed.\n')
 
 
+class OpenShiftAnsibleSyntaxCheck(Command):
+    ''' Command to run generated module validation'''
+    description = "Run generated module validation"
+    user_options = []
+
+    def initialize_options(self):
+        ''' initialize_options '''
+        pass
+
+    def finalize_options(self):
+        ''' finalize_options '''
+        pass
+
+    # self isn't used but I believe is required when it is called.
+    # pylint: disable=no-self-use
+    def run(self):
+        ''' run command '''
+
+        has_errors = False
+
+        for yaml_file in find_files(os.getcwd(), None, None, r'\.ya?ml$'):
+            with open(yaml_file, 'r') as contents:
+                for line in contents:
+                    # initialize_groups.yml is used to identify entry point playbooks
+                    if re.search(r'initialize_groups\.yml', line):
+                        print('-' * 60)
+                        print('Syntax checking playbook: %s' % yaml_file)
+                        try:
+                            subprocess.check_output(
+                                ['ansible-playbook', '-i localhost,',
+                                 '--syntax-check', yaml_file]
+                            )
+                        except subprocess.CalledProcessError as cpe:
+                            print('Execution failed: %s' % cpe)
+                            has_errors = True
+        if has_errors:
+            print('Ansible syntax-check issues found')
```

Comment:
I ran this test (`python setup.py ansible_syntax`) and got the error message above ^

**nit**: `Ansible syntax-check issues found` is a pretty mundane looking output message. Could we prefix that with **ERROR:** or something? I almost missed it (though, there were obvious errors printed in the messages above that lint).

I won't block on it though.